modules/configuring-firewall.adoc

```diff
@@ -75,7 +75,10 @@ Before you install {product-title}, you must configure your firewall to grant ac
 |URL | Function
 
 |`mirror.openshift.com`
-|Required to access mirrored installation content and images
+|Required to access mirrored installation content and images.  Also a source of release image signatures, although the cluster-version operator needs only a single functioning source.
```

Comment:
```suggestion
|Required to access mirrored installation content and images. Although the Cluster Version Operator requires only a single functioning content source, it also is a source of release image signatures.
```

Reply:
I merged your suggestion, and then took a closer look while rebasing onto master.  I've since walked back most of the changes, but kept your "Cluster Version Operator".  My main concern with your:

> Although the Cluster Version Operator requires only a single functioning content source, it also is a source of release image signatures.

is that "it" reads as "the Cluster Version Operator", when it's the mirror that's the source of signatures.  Points I want to get across in this entry:

* `mirror.openshift.com` is a source of installation content and images (covered in existing wording that predates this PR).
* `mirror.openshift.com` is a source of release image signatures (new in this PR).
* The CVO only needs a single, functioning source of signatures.  So for this purpose you need `mirror.openshift.com` or `storage.googleapis.com/openshift-release`, but you don't *need* both (it's fine to allow both if you want).  (new in this PR)

I'm certainly open to other phrasing attempts that cover all three points, if you still think my current wording is unclear.